src/ts/components/listboxtogglebutton.ts

```diff
@@ -0,0 +1,68 @@
+import {ToggleButton, ToggleButtonConfig} from './togglebutton';
+import {ListBox} from './listbox';
+import {UIInstanceManager} from '../uimanager';
+
+export interface ListBoxToggleButtonConfig extends ToggleButtonConfig {
```

Comment:
Continuing the comment from above, if the `ListBox` is a child of a `SettingsPanel`, there's also no need for this component but the already existing `SettingsToggleButton` will be used. Of course there can be a `SubtitleSettingsToggleButton` if a button with extended functionality specifically for the subtitle settings panel is required.